src/API/Site/Controllers/MerchantCenter/ShippingRateController.php

```diff
@@ -317,28 +318,52 @@ protected function get_all_shipping_rates(): array {
 	protected function get_shipping_rates_for_country( string $country ): array {
 		$results = $this->create_query()->where( 'country', $country )->get_results();
 
-		return $this->group_rates_by_country( $results );
+		return $this->prepare_country_rates_for_output( $results );
 	}
 
 	/**
+	 * Prepares the given shipping rates array and returns them in the format used by the API.
+	 *
 	 * @param array $rates
 	 *
 	 * @return array
 	 *
 	 * @since x.x.x
 	 */
+	protected function prepare_country_rates_for_output( array $rates ): array {
+		$rates_output = [];
+		foreach ( $rates as $rate ) {
+			// We don't render the class rates because they are not used by the API.
+			if ( isset( $rate['options']['shipping_class_rates'] ) ) {
```

Comment:
💅 

I think the `if conditional` could be remove and we just have ` unset( $rate['options']['shipping_class_rates'] ); `
```
			if ( isset( $rate['options']['shipping_class_rates'] ) ) {
				unset( $rate['options']['shipping_class_rates'] );
			}
```

Reply:
Done. https://github.com/woocommerce/google-listings-and-ads/pull/1192/commits/aff1101e9655f8ae61ce0e4799117955f7747fc9

Reply:
Excellent thanks! 👍